docs/markdown/Getting-meson.md

```diff
@@ -60,7 +60,7 @@ module](Python-module.md) and do not have any external dependencies
 will continue to work as expected.
 
 Please note that this is a new feature, so bug reports are expected
-and welcome!
+and reports are welcome!
```

Comment:
This does not seem to fix a typo but instead introduces a stylistic error, because the word "reports" is now duplicated.

Reply:
Agreed. IMHO this PR can be closed.

If I were going to make changes, I'd do something like this:

Please note that this is a new feature, so bug reports and feedback are welcome.

Reply:
Sorry, i meant to fix it so it would say "Please note that this is a new feature, so bugs are expected and reports are welcome!", because it feels like its saying that bugs are welcome.
Its a minor detail, if you want me to fix it (correctly this time) please lmk!